src/afl-fuzz-run.c

```diff
@@ -44,6 +58,13 @@ u8 run_target(afl_state_t *afl, u32 timeout) {
      territory. */
 
   memset(afl->fsrv.trace_bits, 0, MAP_SIZE);
+  memset(&timer_signal_event, 0, sizeof(struct sigevent));
+
+  timer_signal_event.sigev_notify = SIGEV_THREAD;
+  timer_signal_event.sigev_notify_function = timeout_handle;
+  timer_signal_event.sigev_value.sival_ptr = &afl;
+  timer_create(CLOCK_REALTIME, &timer_signal_event, &timer);
```

Comment:
I think we should use `CLOCK_MONOTONIC` for this use case, see https://stackoverflow.com/a/3527632/1345238
Then we may even be able to do without `-lrt` but I'm not sure.